app/controllers/application_controller.rb

```diff
@@ -80,6 +80,10 @@ def failed_update_error(obj, obj_name)
   def failed_destroy_error(obj, obj_name)
     "#{_('Could not delete the %{o}.') % {o: obj_name}} #{errors_to_s(obj)}"
   end
+  
+  def success_message(obj_name, action)
```

Comment:
Yes I agree. In standard Rails weirdness though, helpers are available to views but not easily available to controllers so the old standard was to place anything shared by controllers into the base application_controller. 
It looks like they have switched to something called 'concerns' instead as of Rails 4 which seems to play the role of a helper module. I'll read up on it and switch some of the functions in the application_controller over. We should be moving in that Rails 4+ direction.